init.sh

```diff
@@ -1,4 +1,8 @@
-KEY="mykey"
+#!/bin/bash
```

Comment:
Is there any good reason not to add this shebang, which the linter has asked me to?

Reply:
No I think it's good practice